File: 3405-CountTheNumberOfSpecialCharactersIi/3405-CountTheNumberOfSpecialCharactersIi.cpp
// Last updated: 20/07/2026, 11:25:15
class Solution {
public:
    int numberOfSpecialChars(string word) {
        int count = 0;
        for(char i = 'a'; i <= 'z'; i++) {
            int lastLower = word.rfind(i);
            int firstUpper = word.find(toupper(i));
            if(lastLower != -1 && firstUpper != -1 && lastLower < firstUpper) {
                count++;
            }
        }
        return count;
    }
};

File: 3405-CountTheNumberOfSpecialCharactersIi/3405-CountTheNumberOfSpecialCharactersIi.py
# Last updated: 20/07/2026, 11:25:20
class Solution:
    def numberOfSpecialChars(self, word: str) -> int:
        count=0
        for i in 'abcdefghijklmnopqrstuvwxyz':
            last_lower=word.rfind(i)
            first_upper=word.find(i.upper())
            if last_lower!=-1 and first_upper!=-1:
                if last_lower<first_upper:
                    count+=1
        return count